plugins/aria-visualizer/index.js

```diff
@@ -0,0 +1,104 @@
+/**
+ * Allows users to see what screen readers would see.
+ */
+
+let Plugin = require("../base");
+
+let annotate = require("../shared/annotate")("roles");
+
+// this will let us get a shorter info panel that just
+// lets the user know we are tracking their focus
+const PANEL_OPTIONS = {
+  statusPanelView: true
+};
+
+const ATTRIBUTES = [
+    "role",
+    "aria-hidden"
+];
+
+require("./style.less");
+
+const formatAttributeForMethodName = (attribute) => {
+    return attribute.split("-").map(part => `${part[0].toUpperCase()}${part.substr(1)}`).join("");
+}
+
+class AriaVisualizer extends Plugin {
+    constructor(...args) {
+        const options = Object.assign({}, args, { panel: PANEL_OPTIONS });
+
+        super(options);
+
+        this.ariaAttributes = ATTRIBUTES.reduce((functionMap, attribute) => {
+            const methodSuffix = formatAttributeForMethodName(attribute);
+
+            functionMap[attribute] = {
```

Comment:
There's probably a much better way to handle this, but I wasn't sure how we'd deal with all the various different approaches you might need to take to visualizing the semantics of some of the `aria-*` properties.